docs/consume-packages/Overview-and-Workflow.md

```diff
@@ -36,4 +36,54 @@ Occasionally it's necessary to reinstall packages that are already included in a
 
 Finally, NuGet's behavior is driven by `Nuget.Config` files. Multiple files can be used to centralize certain settings at different levels, as explained in [Configuring NuGet Behavior](../consume-packages/configuring-nuget-behavior.md).
 
+## Ways to install a NuGet Package
+
+NuGet packages are downloaded and installed using any of the methods in the following table. The package may be retrieved from a cache instead of downloaded.
+
+| Tool | Description |
+| --- | --- |
+| [dotnet.exe CLI](install-use-packages-dotnet-cli.md) | (All platforms) CLI tool for .NET Core and .NET Standard libraries, and for SDK-style projects that target .NET Framework (see [SDK attribute](/dotnet/core/tools/csproj#additions)). Retrieves the package identified by \<package_name\>, expands its contents into a folder in the current directory, and adds a reference to the project file. Also retrieves and installs dependencies. |
+| Visual Studio | (Windows and Mac) Provides a UI through which you can browse, select, and install packages and their dependencies into a project from a specified package source. Adds references to installed packages to the project file.<ul><li>[Install and manage packages using Visual Studio](../tools/package-manager-ui.md)</li><li>[Including a NuGet package in your project (Mac)](/visualstudio/mac/nuget-walkthrough)</li></ul> |
+| [PowerShell in Visual Studio](../tools/package-manager-console.md) | (Windows only) Retrieves and installs the package identified by \<package_name\> from a selected source into a specified project in the solution, then adds a reference to the project file. Also retrieves and installs dependencies. |
+| [nuget.exe CLI](install-use-packages-dotnet-cli.md) | (All platforms) CLI tool for .NET Framework libraries and non-SDK-style projects that target .NET Standard libraries. Retrieves the package identified by \<package_name\> and expands its contents into a folder in the current directory; can also retrieve all packages listed in a `packages.config` file. Also retrieves and installs dependencies, but makes no changes to project files or `packages.config`. |
+
+## What happens when a package is installed?
+
+Simply said, the different NuGet tools typically create a reference to a package in the project file or `packages.config`, then perform a package restore, which effectively installs the package. The exception is `nuget install`, which only expands the package into a `packages` folder and does not modify any other files.
+
+The general process is as follows:
+
+1. (All tools except `nuget.exe`) Record the package identifer and version into the project file or `packages.config`.
+
+2. Acquire the package:
+   - Check if the package (by exact identifer and version number) is already installed in the *global-packages* folder as described on [Managing the global packages and cache folders](managing-the-global-packages-and-cache-folders.md).
```

Comment:
the global packages folder an implementation detail.

For any tool, I should just specify the Id, version, source (if not specified, use default), project (this is the project the package reference should be added to).

Reply:
@karann-msft Per discussion, this is moved out of Overview. I'm a little fuzzy on this change, if you still want it. Can you clarify?

Reply:
We'll leave the content changes for after the ToC changes. For now, just moving out is good.